guacamole-docker/bin/start.sh

```diff
@@ -394,6 +417,10 @@ END
     set_optional_property "ldap-max-referral-hops"  "$LDAP_MAX_REFERRAL_HOPS"
     set_optional_property "ldap-operation-timeout"  "$LDAP_OPERATION_TIMEOUT"
 
+    if [ -n "$LDAP_SSL_CERT_FILE" ]; then
```

Comment:
I'm on the fence about this - I certainly don't have any problem supporting this functionality, but I'm not sure its entirely in scope of this particular change??